### DISCUSSION TITLE
How to contribute to elementary OS

### DISCUSSION THREAD

**Discussion by igordsm** [elementary/edw#28] [Accepted Talks]
Following #13, the idea of this talk is to do a "live replay" of one (or two?) merged PRs ([camera/138](https://github.com/elementary/camera/pull/138) or [code/888](https://github.com/elementary/code/pull/888/files)). I would go through the following steps: 

1. getting the source code and compiling our own version of an app
3. exploring source code and finding where in the code things break
4. proposing a first fix ("live" coding)
5. discussing improvements suggested during peer review. 

In a way, it's a look behind the scenes from the POV of a (not so frequent) external contributor. I would comment on both technical and personal experiences when contributing to elementary. Hopefully this would inspire other developers to contribute to this awesome project.

**Comment by jalcine** [comment]
This kind of content would be not only super helpful for new and seasoned developers but it can serve as a living "reference" (if it's kept up to date) for people!

**Comment by cassidyjames** [comment]
@igordsm I'm not sure if you saw, but we've accepted your talk. :) You can get started writing it now if you'd like, and we will provide further instructions/guidance around the recording once we've accepted all the talks.
